polygeist/tools/cgeist/Test/Verification/sycl/single_task.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: sycl-clang.py %s -S -emit-llvm -o %t.ll
+// Test that the LLVMIR generated is verifiable.
+// RUN: opt -verify -disable-output < %t.ll
+// RUN: cat %t.ll | FileCheck %s
+
+// Test that the kernel named `kernel_single_task` is generated.
+// CHECK: define {{.*}}kernel_single_task
+// Test that all referred sycl header functions are generated.
+// CHECK-NOT: declare.*sycl
```

Comment:
```suggestion
// CHECK-NOT:  declare {{.*}} spir_func
```

Reply:
I can do that, but the disadvantage is if we didn't put `spir_func` attribute to a sycl function by mistake, then we don't catch that.

Reply:
Agree. The advantage is that the proposed change gets away with relying on a string. IMO the advantage outweights the disadvantage.